### ISSUE TITLE
Meta issue for workshops not working with Node >4 on Windows

### ISSUE THREAD

**Issue by SomeoneWeird** [nodeschool/discussions#1448]
There is currently a problem with workshops that prevent menu selection with node >4.x.x on windows. I'm creating this issue so we can track the current progress of it.

**Comment by LimitedWard** [comment]
Has there been any update on a workaround for this issue?

**Comment by TylerMoeller** [comment]
If you launch the workshop several times in a row while banging on the keyboard as it launches, arrow keys will eventually work. It's an inconsistent and unreliable workaround though, and you'll have to do it every time you want to run a different lesson. 

For now, you'll have to look at the problems manually in `%APPDATA%\npm\nod_modules\workshop-name\exercises\exercise-name\problem.md`

You can still run and verify your solutions despite the menu issues.

terminal-menu from @substack is used by many of the workshops. I tried the terminal-menu example code and it also fails on Windows 10. https://github.com/substack/terminal-menu#example

I'm seeing these errors after canceling out of the menu:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: write after end
    at writeAfterEnd (C:\node_modules\terminal-menu\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:144:12)
    at DestroyableTransform.Writable.write (C:\node_modules\terminal-menu\node_modules\through2\node_modules\readable-stream\lib\_stream_writable.js:192:5)
    at Stream.ondata (stream.js:31:26)
    at emitOne (events.js:82:20)
    at Stream.emit (events.js:169:7)
    at Stream.Charm.write (C:\node_modules\terminal-menu\node_modules\charm\index.js:95:10)
    at Stream.Charm.reset (C:\node_modules\terminal-menu\node_modules\charm\index.js:111:10)
    at Menu.ondata [as _ondata] (C:\node_modules\terminal-menu\index.js:207:24)
    at DestroyableTransform._transform (C:\node_modules\terminal-menu\index.js:50:18)
    at DestroyableTransform.Transform._read (C:\node_modules\terminal-menu\node_modules\through2\node_modules\readable-stream\lib\_stream_transform.js:184:10)```

**Comment by prijindal** [comment]
This is a problem with the module **terminal-menu**
I have tested it as a standalone package and it was not working

i already opened an issue substack/terminal-menu#34

**Comment by stretch** [comment]
fix it, fix it, fix it, fix it, fix it...

**Comment by martinheidegger** [comment]
I mentioned it in the bug at terminal menu and I mention it here again: this is a problem of node, not terminal window. We found a bugfix but it has not made it to a release yet. Until then: the workshoppers are usable without a menu: just enter 'learnyounode select 0' to select the first exercise (should work for the other workshoppers as well) also you should be able to get a list of entries using 'learnyounode list'

**Comment by jackson-jackson** [comment]
Thanks!

**Comment by martinheidegger** [comment]
Note: A fix will land in the next release of node: https://github.com/nodejs/node/pull/3490#issuecomment-154881552

**Comment by martinheidegger** [comment]
Note: This still means that a lot of people with intermediate versions of node will not be able to run many workshoppers. It would be good to know which terminal / windows combinations are affected from this issue!

**Comment by VishalSubramanyam** [comment]
This issues also exists on Node 5.0.0. I tried many workshoppers with Node 5.0.0, but none of them worked. Many beginners are struggling with this issue, Hope this will get fixed soon.

**Comment by TylerMoeller** [comment]
Verified as fixed with Node v5.1.0. All menus working as expected on my Win10 64bit install. Thanks for getting this checked in. Windows Nodeschoolers will be much happier now.

**Comment by anvis2000** [comment]
I am seeing the same issue still with node version 5.6.0.
